keyboards/handwired/scottokeebs/scotto9/info.json

```diff
@@ -0,0 +1,47 @@
+{
+    "manufacturer": "ScottoKeebs",
+    "keyboard_name": "Scotto9",
+    "maintainer": "joe-scotto",
+    "bootloader": "caterina",
```

Comment:
```suggestion
```

Caterina bootloader is implied with ["promicro" development board](https://github.com/qmk/qmk_firmware/blob/master/data/mappings/defaults.hjson#L3-L7).

Can you proactively update this in your current active PRs?